src/main/resources/templates/admin/moduleView.html

```diff
@@ -0,0 +1,535 @@
+<!DOCTYPE HTML>
+<html xmlns:th="http://www.thymeleaf.org">
+<head th:replace="~{/admin/header :: head}"/>
+<body>
+
+<span th:replace="~{/admin/header :: top}"/>
+
+<a href="/admin/">Home</a>
+-
+<a href="/admin/modules">Modules</a>
+
+<th:block th:if="${canManageModules}">
+    <h2 th:text="|Module ${module.name}|">Module (name)</h2>
+
+    <p>
+        Preview module content and schedule events. Event schedules are all in UTC. CMD or CRTL +
+        click to select multiple projects for adding/removing from events
+    </p>
+
+    <h3> Cohorts and Projects </h3>
+    <div th:each="cohort : ${module.cohorts}">
+        <h4 th:text="${cohortNames.get(cohort.cohortId)} + ' '
+            + ${#temporals.format(cohort.startDate, 'yyyy/MM/dd')} + ' - '
+            + ${#temporals.format(cohort.endDate, 'yyyy/MM/dd')}">
+            Cohort Name (Start Date - End Date)
+        </h4>
+        <ul th:each="projectId : ${cohort.projects}">
+            <li th:text="${projectNames.get(projectId)}"> Project Name </li>
+        </ul>
+    </div>
+
+
+    <th:block th:if="${module.overview}">
+        <hr>
+        <h3> Overview </h3>
+        <div data-th-utext="${module.overview}"> </div>
+    </th:block>
+
+    <th:block th:if="${module.preparationMaterials}">
+        <hr>
+        <h3> Preparation Materials </h3>
+        <div data-th-utext="${module.preparationMaterials}"> </div>
+    </th:block>
+
+    <th:block th:if="${module.additionalResources}">
+        <hr>
+        <h3> Additional Resources </h3>
+        <div data-th-utext="${module.additionalResources}"> </div>
+    </th:block>
+
+    <th:block th:if="${oneOnOneInfo}">
+        <hr>
+        <h3> 1:1 Sessions </h3>
+        <div data-th-utext="${oneOnOneInfo}"> </div>
+
+        <h4> Sessions </h4>
+        <table>
+            <thead>
+            <tr>
+                <th>ID</th>
+                <th>Start Time</th>
+                <th>End Time</th>
+                <th>Meeting Link</th>
+                <th>Recording Link</th>
+                <th>Slides Link</th>
+                <th>Remove projects</th>
+                <th>Add projects</th>
+                <th></th>
+                <th></th>
+            </tr>
+            </thead>
+
+            <tbody>
+            <tr th:each="event : ${oneOnOneEvents}" class="striped">
+                <td th:text="${event.id}">Event ID</td>
+                <td>
+                    <input type="datetime-local"
+                           name="startTime"
+                           required
+                           th:form="|update-${event.id}|"
+                           th:value="${#temporals.format(event.startTime.atZone('UTC'), dateFormat)}" />
```

Comment:
Is it going to be asking for trouble to make people do the conversions between UTC and local timezones themselves? I could see that being a common source of user error, but maybe people are already used to doing it and it's not actually a big deal.

The alternative, I guess, would be to put a time zone selector on the page.

Reply:
Opting to keep it in UTC for now for simplicity. 

If product team likes to have this feature we can add it then. Likely we can expand this to show times in project timezones as a separate column as well. But there will be a lot more data interdependencies and more javascript.